bluer-tools/src/blumon.rs

```diff
@@ -13,9 +13,92 @@ use std::{
     io::stdout,
     iter,
     time::{Duration, Instant},
+    error::Error,
 };
+
 use tokio::time::sleep;
 
+use chrono::{DateTime, Utc};
+use serde::{Serialize, Deserialize};
+use serde_jsonlines::append_json_lines; //https://jsonlines.org/
+use clap::Parser;
+
+#[derive(Parser, Debug)]
+pub struct Opts {
+    /// The filename to write the advertisement report log to in json lines format
+    #[clap(short, long, default_value = "", help="The filename to write the advertisement report log to in json lines format")]
```

Comment:
```suggestion
    #[clap(short, long)]
```
Clap should use the docstring as help anyway.